Fix a failing test on Windows

cdelmas opened the pull request:
Hi @jan-molak,
Thank you for the patch. There is still a failing test, but I guess you don't have a Windows environment to test it, so I give you this little patch that makes the project build on Windows platform (issue #3).
The project should still build anywhere else (didn't try though).

cdelmas committed:
Fix a failing test on Windows

jan-molak commented:
Thanks for a PR, interesting! I've got it passing on Windows using [this custom matcher](https://github.com/smartcodeltd/release-candidate-maven-plugin/blob/master/src/test/java/com/smartcodeltd/matchers/EqualsIgnoringOSSpecificLineSeparators.java). Is `mvn clean verify` still failing for you when using the latest code?

cdelmas commented:
Yes, you use this matcher for VersionMojoTest (and that works well), but I still have a failing test (com.smartcodeltd.UpdateVersionMojoTest#makes_the_version_the_only_difference_between_updated_and_original_pom) because of the hardcoded "\n" in the Difference class. That's the point of my PR; maybe you can fix it otherwise.

cdelmas committed:
Do not change version of organization poms.

Change the root project search method to ignore organization poms.
Select a parent version to change if the pom is not an organization pom and has the same groupId than the current project and has a snapshot version.

Relates to #5.

jan-molak commented:
I think both PRs got merged into one, but that's OK because they both look good :smile: 
Thanks for your help!
